server/shadow/shadow_encoder.c

```diff
@@ -130,9 +130,6 @@ int shadow_encoder_uninit_grid(rdpShadowEncoder* encoder)
 
 int shadow_encoder_init_rfx(rdpShadowEncoder* encoder)
 {
-	rdpContext* context = (rdpContext*) encoder->client;
-	rdpSettings* settings = context->settings;
-
 	if (!encoder->rfx)
```

Comment:
Just wondering how encoder->rfx can be NULL when we fail if rfx_context_new fails (some lines below)...